microsoft-edge/webview2/how-to/add-host-object-to-script.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Use app objects in JavaScript
+description: Use host object in JavsScript using the WebView2 AddHostObjectToScript API.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 2/17/2022
+---
+# Use app objects in JavaScript
```

Comment:
```suggestion
# Using host objects
```
@jasonstephen15 - change top-of-article title?  if so, change line 2 as well, and TOC title in this PR